ci/do_ci.sh

```diff
@@ -181,25 +219,27 @@ function run_ci_verify () {
 CI_TARGET=$1
 shift
 
+if [[ "$CI_TARGET" =~ bazel.* ]]; then
+    ORIG_CI_TARGET="$CI_TARGET"
+    CI_TARGET="$(echo "${CI_TARGET}" | cut -d. -f2-)"
+    echo "Using \`${ORIG_CI_TARGET}\` is deprecated, please use \`${CI_TARGET}\`"
```

Comment:
We'll probably need to update many docs here :)

Reply:
yeah - i was gonna follow up on that - its already quite out of sync, _and_ the old way still works